Core/CoreTests/Grades/GradeListInteractorLiveTests.swift

```diff
@@ -337,98 +337,6 @@ class GradeListInteractorLiveTests: CoreTestCase {
         subscription.cancel()
     }
 
-    func testShowGradeLetter() {
```

Comment:
The three test cases were deleted because of, calling the updateGradingPeriod in GradeListViewModel in place of GradeListInteractorLive